### PR TITLE
Fix instructions about waffle-tomcat JAR placement

### PR DESCRIPTION
**Pull request by alshain**
See #371

### CODE REVIEW

**Comment by AriSuutariST** [comment]
Hi,

I think it is not possible to sprinkle Waffle jars at same installation into tomcat/lib and WEB-INF/lib. If one wants to use tomcat valves all jars should go to tomcat/lib. Without valves, either directory is ok as long as all needed jars are in that directory (because a class loaded by tomcat/lib classloader won't be able to see classes available in webapp classloader).

Or maybe I just misunderstood the new wording in document ?

**Comment by alshain** [comment]
I will check again whether that works. The other libraries are already
contained in the demo's WEB-INF/lib, that's why assumed they wouldn't have
to be in Tomcat's lib as well,but I'll check.

**Comment by alshain** [comment]
You're correct, the other JARs need to bin in tomcat/lib as well.

**Comment by alshain** [comment]
I have adjusted the documentation.

Now, these JARs are also contained in the war file. They should probably be removed from there.

**Comment by AriSuutariST** [comment]
Yes, you are correct about removing jars from .war file, they are not needed (at least for this case what the document is about, TomcatMixedSingleSignOnAndFormAuthenticatorValve). Maybe this document should instruct only to place them into tomcat/lib.

There seems to be similar text in TomcatSingleSignOnValve.md also.

**Comment by dblock** [comment]
@AriSuutariST this PR is all yours since you seem to understand what goes where (I am not pretending to :), merge at will!

File: Docs/tomcat/TomcatMixedSingleSignOnAndFormAuthenticatorValve.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ Configuring Tomcat
 
 The following steps are required to configure Tomcat with Waffle Mixed Authenticator. 
 
-Package Waffle JARs, including `waffle-jna-1.8.1.jar`, `guava-19.0.jar`, `jna-4.2.2.jar`, `jna-platform-4.2.2.jar`, `slf4j-1.7.21.jar` and `waffle-tomcat[tomcat version].jar` in the application's `lib` directory or copy them to your web server's lib. If you are using Eclipse, you can see which files tomcat is importing by going to Java Recources: `src / Libraries / Apache Tomcat vx.x`. If you've placed it in the tomcat directory and still don't see it, restart Eclipse.
+Package Waffle JARs including `waffle-jna-1.8.1.jar`, `guava-19.0.jar`, `jna-4.2.2.jar`, `jna-platform-4.2.2.jar` and `slf4j-1.7.21.jar` in the application's `WEB-INF\lib` directory.
```
Comment (AriSuutariST) [inline Docs/tomcat/TomcatMixedSingleSignOnAndFormAuthenticatorValve.md]: I would just remove this sentence as it is not relevant to valves.

**Pull request merged**